src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,46 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import React from 'react';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
getAll always returns an array
```suggestion
  const centuries = searchParams.getAll('centuries');
```